### PR TITLE
Disable privacy feature

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
The privacy feature is still under review. This PR will disable it until further notice.

# Solution
Add feature flag to temporarily disable the privacy screen.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [x] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [x] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://318-pr-onfido-sdk-ui-onfido.surge.sh

File: src/locales/en.js [hunk 1]
```diff
@@ -130,15 +130,15 @@ export const en = {
...
-  privacy: {
-    title: 'You\'ll have to upload a photo of your identity document',
-    item_1: 'All your document details must be visible',
-    item_2: 'Your document must be in colour',
-    item_3: 'Avoid light reflections',
-    small_print: 'By continuing, you agree to the <terms>Onfido Terms of Use</terms> and understand that your information, including your facial identifiers, will be processed in accordance with the <privacy>Onfido Privacy Policy</privacy>',
-    decline: 'Decline',
-    continue: 'Continue'
-  },
+  // privacy: {
```
Comment (rfreitas) [inline src/locales/en.js]: this is still commented, how will it work if you the flag is set to true?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://318-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
i have tested the latest surge link. Looks good. So happy to accept this ticket if @rfreitas you can review the latest PR changes :)

**Pull request merged**